tests/tls_e2e/README.md

```diff
@@ -0,0 +1,34 @@
+Unittests Transport Layer Security end-2-end scenarios
+=====================
+
+Test scenarios:
+
+In this test, we have three test hooks for enabling test scenarios.
+
+- fail_cert_verify_callback
```

Comment:
Perhaps, numbering them as 1), 2) and 3) makes sense

Reply:
addressed